addons/mail/models/mail_thread.py

```diff
@@ -4379,9 +4379,18 @@ def _notify_thread_with_out_of_office(self, message, recipients_data, msg_vals=F
 
 
 
+        # limit number of real author / recipient exchanges to 1 every 4 days
+        sent_su = self.env['mail.message'].sudo().search([
+            ('author_id', 'in', ooo_users.partner_id.ids),
+            ('message_type', '=', 'out_of_office'),
+            '|', ('partner_ids', 'in', recipient.ids), ('outgoing_email_to', '=', email_to),
```

Comment:
So it will not send it if just of the recipients already got the message? I guess that's usually fine (probably there's only 1 recipient)